ethcore/src/snapshot/service.rs

```diff
@@ -584,10 +585,19 @@ impl Service {
 	}
 
 	/// Feed a chunk of either kind. no-op if no restoration or status is wrong.
```

Comment:
Maybe update the doc-string to "Feed a chunk of either Block or Status kind. No-op if … …"?